app/lib/plugins.js

```diff
@@ -137,7 +146,7 @@ Plugins.prototype.getAssetPath = function(url) {
  *
  * @return {Array<String>} plug-in paths
  */
-function findPluginEntries(paths) {
```

Comment:
Why rename this one? I renamed it for clarity but it seems like it is not actually clear.

The method returns a list of path to the plugin entry files (i.e. `plugins/foo/index.js`). A good name should make that clear. Otherwise, what is a `pluginPath`? Could be anything :thinking:.

Reply:
We are also talking about entry points to the plug-ins (specifically `index.js` files). But if that is the clear way to go, please rename entry [in this spot](https://github.com/camunda/camunda-modeler/pull/1143/files/3b3e9d0bb974cbde1ebf707b0263b3587412c987#diff-5c671ceac03653fffc86e1ba011735dcR29), too.